src/vaadin-menu-bar-buttons-mixin.html

```diff
@@ -82,6 +88,21 @@
       }
     }
 
+    __updateButtonsTheme() {
+      Array.from(this.shadowRoot.querySelectorAll('vaadin-menu-bar-button'))
```

Comment:
note: you can use `this._buttons.forEach`

Reply:
Updated.